grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy

```diff
@@ -107,34 +110,46 @@ class IndicatorDataService {
         return indicatorData;
     }
 
-    DataGraph getSentStockMovements(def location) {
+DataGraph getSentStockMovements(def location, def params){
+    Integer querySize = params.querySize? params.querySize.toInteger() : 5
+    List allLocations = dataService.executeQuery("select id from openboxes.location")
```

Comment:
Why sql query, cant you use gorm?

Reply:
Don't use schema (`openboxes.`) and I agree with Artur that we should use GORM where possible. If you guys aren't familiar then I can try to write the queries.